examples/example-node-sled-lib.js

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env node
+const assert = require('assert');
+const cp = require('child_process');
+
+// So we don't make real requests to AWS APIs, set ENV=testing
+const env = Object.create(process.env);
+env.ENV = 'testing';
+
+// WIP Integration Test for loadNestedEvent - will fail with "Lookup error:
+// 'events'".
+// At the moment localstack doesn't support step functions so there is
+// no way to do a complete integration test :(
+var child = cp.spawn('./cumulus-sled', ['loadNestedEvent'], { env: env });
+
+child.stderr.pipe(process.stderr);
+
+// example event object
+// TODO(aimee): Write and load these from a file.
```

Comment:
I'm working on this change (reading data from a file in /examples/) now

Reply:
Addressed in https://github.com/cumulus-nasa/cumulus-sled/pull/2/commits/d528aa987d56931d2d0fdd6493c5c84e764162e1 and https://github.com/cumulus-nasa/cumulus-sled/pull/2/commits/7c3c614c5b101a24241a465a8d98ac8278c9493c